web/src/documents.ts

```diff
@@ -76,7 +76,7 @@ export async function close(
   documentId: DocumentId
 ): Promise<Document> {
   return client.call('documents.close', {
-    id: documentId,
+    documentId,
```

Comment:
These changes to JSON-RPC parameter names need corresponding changes to the names on the server-side e.g. https://github.com/stencila/stencila/blob/80e13023befcfdc6bb579f2dd4388a807fa28ca4/rust/src/rpc.rs#L283